### ISSUE TITLE
[BUG] Cloudflare version 2 captcha

### ISSUE THREAD

**Issue by imadarsh1001** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#272]
**Describe the bug**
Getting error while loading cookies file.

**Expected behavior**
Exception in redeem courses: Detected a Cloudflare version 2 Captcha challenge, This feature is not available in the opensource (free) version.

**Screenshots*

![image](https://user-images.githubusercontent.com/14936525/115119993-b55d5380-9fc8-11eb-965c-965705d386e4.png)

**Desktop (please complete the following information):**
 - OS: Windows 10

**Comment by cullzie** [comment]
Hey, This is out of our control to be honest. It looks like your IP has been blocked by cloudflare for the moment. I've seen this if you try to run from a cloud service or if you are hitting the service to aggressively. 
Is the issue still persisting? There's not really a fix we can do around this issue. It might be worth documenting in case others see it though

**Comment by fakeid30** [comment]
Should we close this? @cullzie

**Comment by cullzie** [comment]
> Should we close this? @cullzie 

Ye I thinks so. We can add it to the documentation that it can happen but it's outside of our control

**Comment by jonahanzen** [comment]
I was able to solve it !
I was getting the same error.  I am a complete noob as for programming, so hopefully i'm not saying redundant things but here's what i did:

Things to take into consideration:
OS : Linux - Ubuntu.
Browser : Opera.

I noticed i was using adblocker and was not allowing being tracked in opera, the usual stuff, i also was running things in portuguese as it is my native language. (you see, i'm not sure what actually solved it, i just know it solved.)
I changed the site language to english, deleted all the cookies, restarted the browser, then disabled the adblocker and opera privacy/security options. i resetted the udemy_enroller settings just in case and tried it again, and it worked !

So check if somehow you're blocking things and so cloudflare detects you as a "bot" since it can't track you or something.
Hopefully it helps
